wherehows-web/app/components/dataset-authors.ts

```diff
@@ -64,6 +66,20 @@ export default class DatasetAuthors extends Component {
    */
   userLookup: ComputedProperty<UserLookup> = inject();
 
+  /**
+   * If there are no changes to the ownership tab, we want to keep the save button disabled. Rather than
+   * try to compare two sets of prev vs new data, we just have a flag here that short stops the validation
+   * function.
+   * Note: hasChanged has 3 states:
+   * -1   Component hasn't completed initialization yet, and no changes have been made. When
+   *      requiredMinNotConfirmed is run on init/render, this gets incremented to its neutral state at 0
+   * 0    No changes have been made yet
+   * 1    At least one change has been made
+   * @type {number}
+   * @memberof DatasetAuthors
+   */
+  hasChanged: Comparator = -1;
```

Comment:
I'm a bit confused by this, the property name indicates a dual state `has-*` i.e. either it is or it isn't, but the value can have three states -1, 0, 1? Doesn't seem to make sense to me

Reply:
That's true, it was a leftover from originally a boolean until I realized it wouldn't work as a boolean and I needed 3 states. Will update naming to `changedState` or something to make more sense.